.github/workflows/deploy.yaml

```diff
@@ -99,6 +67,7 @@ jobs:
           cf api "${CF_API}"
           cf auth
           cf target -o "${CF_ORG}" -s "${CF_SPACE}"
+
       # Parse app name from manifest to ensure it matches up
       - name: Fetch app name from manifest
         run: echo "APP_NAME=$(ruby -e "require 'yaml'; config = YAML.load_file('manifest.yml'); puts config['applications'][0]['name']")" >> $GITHUB_ENV
```

Comment:
Hm, does this need updating now we've moved the `manifest.yml` file? Although it seems to be working at the moment 🤔

Reply:
Because the manifest file will end up in the root of the working directory either way, it should be fine.